gen/Orientation_Data.c

```diff
@@ -0,0 +1,359 @@
+
+
+/*
```

Comment:
You're right, we can. I've removed them, so it should call `rtiddsgen` and generate files based on the version of Connext being used.

The previous release was for IData 3.x. I think having separate releases so that both options are available is a good idea.